src/game.cpp

```diff
@@ -232,6 +232,8 @@ static const trait_id trait_M_IMMUNE( "M_IMMUNE" );
 static const trait_id trait_PARKOUR( "PARKOUR" );
 static const trait_id trait_VINES2( "VINES2" );
 static const trait_id trait_VINES3( "VINES3" );
+static const trait_id trait_VINES3( "VINES3" );
```

Comment:
This won't compile. You duplicated the line above.

Reply:
Oops, line has been deleted and built locally